build.go

```diff
@@ -48,6 +48,7 @@ type BuildOptions struct {
 	Buildpacks        []string
 	ProxyConfig       *ProxyConfig // defaults to  environment proxy vars
 	ContainerConfig   ContainerConfig
+	FileFilter        build.FileFilter
```

Comment:
We should redefine this type here instead of depending on the use of an internal definition.

Reply:
@jromero when you say redefine are you saying:
```
FileFilter   func(string) bool
```
or add a new type in the file
```
type FileFilter func(string) bool
```